domain/application/errors/errors.go

```diff
@@ -31,6 +31,14 @@ const (
 	// not valid.
 	ApplicationNameNotValid = errors.ConstError("application name not valid")
 
+	// UnitNotFound describes an error that occurs when the unit being operated on
+	// does not exist.
+	UnitNotFound = errors.ConstError("unit not found")
```

Comment:
Do we have any places in facade needs to change for this?

Reply:
No, I replaced all the usages of the original `uniterrors.NotFound ` error.